src/RoleObjects/ProgressBarData.js

```diff
@@ -6,14 +6,14 @@ export default class ProgressData extends RangeData {
   }
 
   get valueMax() {
-    this._reactProps.max;
+    return this._reactProps.max;
```

Comment:
Great example of why linting is a good thing and I probably should have added it to this project long ago!